external/anomaly/ote_anomalib/callbacks/score_report.py

```diff
@@ -0,0 +1,43 @@
+"""Score reporting callback"""
+
+# Copyright (C) 2020 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions
+# and limitations under the License.
+
+from typing import Optional
+
+from ote_sdk.entities.train_parameters import TrainParameters
+from pytorch_lightning import Callback
+
+
+class ScoreReportingCallback(Callback):
+    """
+    Callback for reporting score.
+    """
+
+    def __init__(self, parameters: Optional[TrainParameters] = None) -> None:
+        if parameters is not None:
+            self.score_reporting_callback = parameters.update_progress
+        else:
+            self.score_reporting_callback = None
+
+    def on_train_epoch_end(self, trainer, pl_module):
```

Comment:
I think this can be safely changed back to `on_validation_epoch_end`

Reply:
It's right. I'll change it as you said.